solr_scripts/schema.xml

```diff
@@ -78,10 +76,10 @@
         <filter class="solr.LowerCaseFilterFactory"/>
         <filter class="solr.EnglishPossessiveFilterFactory"/>
         <filter class="solr.KeywordMarkerFilterFactory" protected="protwords.txt"/>
-        <filter class="solr.EnglishMinimalStemFilterFactory"/>
         <!-- Optionally you may want to use this less aggressive stemmer instead of PorterStemFilterFactory:
-          <filter class="solr.PorterStemFilterFactory"/>
+          <filter class="solr.EnglishMinimalStemFilterFactory"/>
```

Comment:
We should change the stemmers back to what they were. It might not make that big of a difference but since just in case, it's good to be consistent.